package.json

```diff
@@ -27,6 +27,7 @@
     "eslint-loader": "^1.0.0",
     "eslint-plugin-babel": "^4.1.2",
     "eslint-plugin-react": "^7.4.0",
+    "file-loader": "^0.11.2",
```

Comment:
Does file loader need to be updated as part of this PR?

Reply:
It  is added because it was missing